lib/Sky/Api.php

```
@@ -365,4 +400,43 @@ public function singular($resource_name)
         return $this->resources[$resource_name]['singular'] ?: substr($resource_name,0,-1);
     }
 
+    /**
+     * Converts a hyphentated string into camelCase
+     * @param string $input hyphenated string
+     * @param string $word_delimiter defaults to '-'
+     * @return string
+     */
+    public function toCamelCase($input, $word_delimiter = '-')
+    {
+        if (!is_string($input)) return false;
```

should throw an InvalidArgumentException if it isn't a string.